baseplate/clients/redis.py

```diff
@@ -4,6 +4,7 @@
 from typing import Optional
 
 import redis
+import rediscluster
```

Comment:
This would make anyone using the current redis client suddenly have to install `redis-py-cluster` as well just to continue using the other client.  Can you move the new cluster stuff into its own module in the clients folder? Like `baseplate/clients/redis_cluster.py` perhaps?